### PR TITLE
Fix relay workflow

### PR DESCRIPTION
**Pull request by vpnu**
## Pull Request Contents

| |
|-|
🦋 Bug Fix

## Description

- Fix broken workflow action for Hutch.Relay release

## Related Issues or other material

- [x] This PR relates to one or more issues, detailed below

- Related Issue #69 

- [x] I've completed all **actions** and **tasks** detailed in the PR Template

### CODE REVIEW

**Review by AndyRae — DISMISSED** [review]

File: .github/workflows/release.Hutch.Relay.yml [hunk 1]
```diff
@@ -10,30 +10,40 @@ on:
 
 env:
   namespace: hutchstack
-  repo-name: hutch-relay
+  image-name: hutch-relay
+  repo-name: ${{ github.repository }}
```
Comment (dcl10) [inline .github/workflows/release.Hutch.Relay.yml:14]: Doesn't look like you're using this variable

Comment (dcl10) [reply .github/workflows/release.Hutch.Relay.yml:14]: Sorry this doesn't make it clear. I mean the `repo-name` one

Comment (vpnu) [reply .github/workflows/release.Hutch.Relay.yml:14]: repo-name and image-name are used on line 46

Comment (dcl10) [reply .github/workflows/release.Hutch.Relay.yml:14]: Oh I see now. I got distracted by the diff highlights lol

**Review by dcl10 — APPROVED** [review]

File: .github/workflows/release.Hutch.Relay.yml [hunk 2]
```diff
@@ -10,30 +10,40 @@ on:
...
       - name: Set timestamp env var
         run: echo "RUN_TIMESTAMP=$(TZ="Etc/UTC" date +'%Y%m%d%H%M%S')" >> $GITHUB_ENV
-      
+
       - name: Docker Metadata action
         id: meta
         uses: docker/metadata-action@v5.5.1
         with:
-          images: ${{ env.namespace }}/${{ env.repo-name }}
+          images: ${{ env.registry }}/${{ env.repo-name }}/${{ env.image-name}}
```
Comment (beforan) [inline .github/workflows/release.Hutch.Relay.yml:46]: I think this is maybe too nested.
(not certain, may depend on repo renaming.)

It currently produces `ghcr.io/health-informatics-uon/rquest-tools/hutch-relay`

I think we want one of the below:
- `ghcr.io/health-informatics-uon/hutch-relay`
- `ghcr.io/health-informatics-uon/hutch/relay`

**Review by beforan — APPROVED** [review]

**Pull request merged**